File: original_scripts/r/bubble_sort/qwen3_8b.r
bubble_sort <- function(x) {
  n <- length(x)
  for (i in 1:n) {
    for (j in 1:(n - i)) {
      if (x[j] > x[j + 1]) {
        temp <- x[j]
        x[j] <- x[j + 1]
        x[j + 1] <- temp
      }
    }
  }
  x
}

File: modified_scripts/scala/bubble_sort/qwen3_8b.scala
def bubbleSort(arr: Array[Int]): Unit = {
  for (i <- 0 until arr.length - 1) {
    for (j <- 0 until arr.length - i - 1) {
      if (arr(j) > arr(j + 1)) {
        val temp = arr(j)
        arr(j) = arr(j + 1)
        arr(j + 1) = temp
      }
    }
  }
}